### ISSUE TITLE
currentColor interaction with PDFkit

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#1182]
The current version of the develop branch generates PDFs (with PDFkit.js) having white staff lines:

<img width="1226" alt="Screen Shot 2019-11-03 at 3 51 07 PM" src="https://user-images.githubusercontent.com/3487289/68094515-3157f000-fe56-11e9-942a-b2eb5b0ae6dc.png">

Is there a way of getting PDFkit to make black staff lines?  Otherwise, the recent changes related to SVG  currentColor should be reverted.  MeasuringPolyphony has black stafflines in the PDFs, but I don't see any options related to currentColor, so perhaps it is using an older version of the verovio toolkit?

**Comment by craigsapp** [comment]
An easy way to solve the problem is to convert the `currentColor` text in the SVG text content into the word `black`:

```javascript
vrvToolkit.renderToSVG(i+1, {}).replace(/\bcurrentColor\b/g, "black")
```

I do that before sending to the PDFkit function `SVGtoPDF()` and that solves the problem:

<img width="1223" alt="Screen Shot 2019-11-03 at 4 45 02 PM" src="https://user-images.githubusercontent.com/3487289/68095008-c8bf4200-fe5a-11e9-9fc7-48d9e7032df4.png">

That is a sufficient solution for me, so you can close this issue and leave the use of currentColor as it is.

**Comment by lpugin** [comment]
Should be fixed with 89250a1d06297d4fb90b7d3c85bd2f1efbb0b81d
